redis/src/script.rs

```diff
@@ -206,12 +206,25 @@ impl<'a> ScriptInvocation<'a> {
         Ok(hash)
     }
 
-    fn load_cmd(&self) -> Cmd {
+    /// Returns a command to load the script.
+    pub fn load_cmd(&self) -> Cmd {
```

Comment:
why does this need to be a public method? the user has the actual script, this uses a single argument from the whole invocation.

Reply:
I decided to make both methods public `load_cmd` and `eval_cmd` so they can be independently added to pipelines just through `add_command`. I can change them to be `pub(crate)` if you don't want them to be exposed.

In theory the `load_cmd` could also be a method of `Script`, I kept it here because all the execution (e.g. `load()`) is on the `ScriptInvocation`.

Reply:
It can be still loaded by using the `ScriptInvocation`, currently using it like this:

```rs
    pub fn invoke(&self, con: &mut Connection<'_>) -> Result<Vec<CardinalityScriptResult>> {
        match self.pipe.query(con) {
            Ok(result) => Ok(result),
            Err(err) if err.kind() == redis::ErrorKind::NoScriptError => {
                self.script.load_redis(con)?;
                self.pipe
                    .query(con)
                    .map_err(relay_redis::RedisError::Redis)
                    .map_err(Into::into)
            }
            Err(err) => Err(relay_redis::RedisError::Redis(err).into()),
        }
    }
```

But it can't be added to the pipeline without it, but that can also be solved by adding another fn to the `implement_commands`.